pulsar-sql/presto-pulsar/src/main/java/org/apache/pulsar/sql/presto/PulsarSplitManager.java

```diff
@@ -229,13 +233,13 @@ public ConnectorSplitSource getSplits(ConnectorTransactionHandle transactionHand
 
     @VisibleForTesting
     Collection<PulsarSplit> getSplitsNonPartitionedTopic(int numSplits, TopicName topicName,
-            PulsarTableHandle tableHandle, SchemaInfo schemaInfo, TupleDomain<ColumnHandle> tupleDomain,
-             OffloadPolicies offloadPolicies) throws Exception {
-        ManagedLedgerFactory managedLedgerFactory = PulsarConnectorCache.getConnectorCache(pulsarConnectorConfig)
-                .getManagedLedgerFactory();
+                                                         PulsarTableHandle tableHandle, SchemaInfo schemaInfo,
+                                                         TupleDomain<ColumnHandle> tupleDomain,
+                                                         OffloadPolicies offloadPolicies) throws Exception {
+        ManagedLedgerFactory managedLedgerFactory = pulsarConnectorCache.getManagedLedgerFactory();
 
         return getSplitsForTopic(
-                topicName.getPersistenceNamingEncoding(),
```

Comment:
Maybe we could keep the old code? it could compatible with the legacy naming scheme.